### PR TITLE
[risk=no]genomic Filters

### PR DESCRIPTION
**Pull request by mhigbyflowers**
image

the next PR will address the sliders

### CODE REVIEW

File: public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx [hunk 1]
```diff
@@ -470,109 +467,127 @@ export const GenomicViewComponent = withRouteData(
...
+                <div className='faq-heading-text'>Questions about genomics?
+                  <span style={styles.faqLink} onClick={() => this.topBarClick(3)}>Learn More</span>
                 </div>
               </div>
             </div>
           </div>
-        </React.Fragment>
-      );
+          <div style={styles.innerContainer} id="childView">
+            {selectionId === 1 && (
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx:500]: Is this duplicated ?

**Review by SrushtiGangireddy — COMMENTED** [review]
image
Looks like we are getting 2 views of the whole panel.

File: public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx (hunk 1 above)
Comment (mhigbyflowers) [reply public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx:500]: Strange I thought I pushed this fix. 
```suggestion
            {selectionId === 1 && (
```

Comment (mhigbyflowers) [reply public-ui/src/app/data-browser/views/genomic-view/genomic-view.component.tsx:500]: i dont even know how I made a 'suggested change' I will fix it

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good to me. Thanks.

**Pull request merged**